packages/react-components/react-toast/src/state/useToaster.ts

```diff
@@ -1,19 +1,22 @@
-import { useFluent_unstable as useFluent } from '@fluentui/react-shared-contexts';
 import * as React from 'react';
 import { Toaster } from './vanilla/toaster';
 import { useForceUpdate } from '@fluentui/react-utilities';
-import { Toast, ToastId, ToastPosition } from './types';
+import { DefaultToastOptions, Toast, ToastPosition, ToasterOptions } from './types';
 
-export function useToaster() {
-  const { targetDocument } = useFluent();
+export function useToaster<TElement extends HTMLElement>(options: ToasterOptions = {}) {
   const forceRender = useForceUpdate();
-  const [toaster] = React.useState(() => {
-    if (targetDocument) {
-      const newToaster = new Toaster(targetDocument);
-      newToaster.onUpdate = forceRender;
-      return newToaster;
+  const defaultToastOptions = useDefaultToastOptions(options);
```

Comment:
```suggestion
  const toastOptions = useToastOptions(options);
```

I don't understand "default" prefix there

Reply:
The idea is that users can configure all toast options, the ones applied to the toaster is the 'default' for all toasts